README.md

```diff
@@ -77,6 +77,12 @@ const totalPrice = 500; // Pass in the total price in smallest currency unit (ie
 const currency = 'USD';
 patch.orders.createOrder({ total_price: totalPrice, currency: currency });
 
+// Create order with the issued_to field (optional)
+const amount = 1_000_000; // Pass in the amount in unit specified
+const unit = 'g';
+const issued_to = { email: 'issuee@companya.com', name: 'Olivia Jones' };
```

Comment:
Can we call this variable `issuedTo` to match the naming convention of the rest of the doc? Thanks!